src/edu/cmu/cs/sasylf/Main.java

```diff
@@ -1,13 +1,5 @@
 package edu.cmu.cs.sasylf;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.PrintStream;
-import java.io.Reader;
-

```

Comment:
It's standard (or was standard) to put all standard imports before imports of app specific classes.  Why reorganize everything?